internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -687,20 +687,43 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				}
 			}
 
+			overrideTypes := [2]string{"categories", "rules"}
```

Comment:
are we able to do it without this and loop the provided nested rules? hardcoding this in takes away part of the automated nature where adding a new block in the terraform provider is automatically picked up in this library.

Reply:
that's a good point. I've updated the code, so it pulls out nested rules automatically